apiserver/facades/agent/uniter/state.go

```diff
@@ -10,9 +10,13 @@ import (
 	"github.com/juju/juju/state"
 )
 
-type storageInterface interface {
+type storageAccess interface {
+	storageInterface
 	VolumeAccess() storageVolumeInterface
```

Comment:
Shouldn't these interfaces be renamed to match the other ones (and the method names)?